### PR TITLE
First commit to the geoMean task

### PR DESCRIPTION
**Pull request by PszemekZ**

### CODE REVIEW

File: geoMean.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Napisz funkcję geoMean(), która zwróci średnią geometryczną podanej tablicy.
+
+function geoMean(array) {
+    let multiples = array[0];
```
Comment (dabrowskisoftware) [inline geoMean.js]: You could set it to 1 in the beginning and then use a normal for loop starting from the first element. But this is ok as well.

Comment (dabrowskisoftware) [reply geoMean.js]: multiples is not a good name for a number. It should be called `product.`

**Review by dabrowskisoftware — COMMENTED** [review]
It's ok but it's a perfect case for Array.reduce:

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

Please, change your solution to use Array.reduce.

File: geoMean.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+// Napisz funkcję geoMean(), która zwróci średnią geometryczną podanej tablicy.
+
+function geoMean(array) {
+  return Math.pow(array.reduce(function(prev, next) {
```
Comment (dabrowskisoftware) [inline geoMean.js]: Everything is ok except one parameter name.

According to this:

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

the first parameter is not a previous element. It's an accumulator.

File: geoMean.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+// Napisz funkcję geoMean(), która zwróci średnią geometryczną podanej tablicy.
+
+function geoMean(array) {
+  return Math.pow(array.reduce(function(acc, next) {
```
Comment (dabrowskisoftware) [inline geoMean.js]: What is acc? Acceptance? Accelerator? Name it with no shortcuts! :)

**Review by dabrowskisoftware — CHANGES_REQUESTED** [review]

File: geoMean.js (hunk 3 above)
Comment (PszemekZ) [reply geoMean.js]: Account 🥇  sorry about that

**Review by dabrowskisoftware — APPROVED** [review]

**Pull request merged**